core/data/tests/can_generate_unit_structs/output.kt

```diff
@@ -0,0 +1,14 @@
+/**
+ * Generated by typeshare 1.0.0
+ */
+
+@file:NoLiveLiterals
+
+package com.agilebits.onepassword
+
+import androidx.compose.runtime.NoLiveLiterals
+import kotlinx.serialization.*
+
+@Serializable
+class UnitStruct
```

Comment:
@InquisitivePenguin 
I would expect this to use Kotlin's `object` syntax like this repo does for enums. This would make it similar to tests `boxed_value` and `can_handle_anonymous_struct`.
```kt
@Serializable
@SerialName("UnitStruct")
object UnitStruct
```

Reply:
Thanks for bringing this up. I can open a PR to follow-up on this once this PR is merged.